packages/language/src/validation/internal-codes.ts

```diff
@@ -21,6 +21,14 @@ export const InternalCodes = {
   } as ParametricPLICode,
 };
 
+export const MacroCodes = {
+  Case: {
+    code: "MACRO0001",
+    severity: Severity.W,
+    message: (token: any) => `Input text "${token}" must be uppercase.`,
+  } as ParametricPLICode,
+};
```

Comment:
Suggestion: Can we move this somewhere else? The internal codes are AFAIK only those used for internal testing purposes.